Sources/BrowserServicesKit/ContentBlocking/UserScripts/TrackerResolver.swift

```diff
@@ -221,7 +221,7 @@ fileprivate extension KnownTracker.Rule {
 private extension KnownTracker.ActionType {
 
     func toTrackerResolverRuleAction() -> TrackerResolver.RuleAction {
-        self == .block ? .blockRequest : .allowRequest
+        self == .block || self == .blockCtlFB ? .blockRequest : .allowRequest
```

Comment:
btw there is inconsistency in naming one you use CTL and in this case there is Ctl, please make sure we use it capitalized everywhere (unless the var starts with ctl, in such case use all lowercased)

Reply:
Yeah please disregard that comment, that's for a previous discussion/rev of the PR.

For the naming issue, this was just landed in https://github.com/duckduckgo/TrackerRadarKit/pull/19, not sure the effort is worth the squeeze here?

If we do want to address it, maybe it makes more sense to follow the more conventional approach of camelCasing acronyms for all instances of CTL (i.e any acronym of 3+ chars).  That would better match the rest of our code I think?  Or I can also update the TRK PR, just LMK which way you prefer.

Reply:
We got a rule (that is obviously not followed everywhere) https://github.com/duckduckgo/swift?tab=readme-ov-file#capitalize-acronyms - so it would be awesome to fix it

Reply:
@jaceklyp cool, fixed!  FWIW I don't have access to that repo (/swift), any reason it can't be public?  Seems like something that would benefit from broad visibility.

Reply:
Related TRK PR: https://github.com/duckduckgo/TrackerRadarKit/pull/22